internal/backends/winit/event_loop.rs

```diff
@@ -512,13 +526,27 @@ pub fn run() {
 
         Event::NewEvents(_) => {
             *control_flow = ControlFlow::Wait;
+
+            windows_with_pending_redraw_requests.clear();
+            ALL_WINDOWS.with(|windows| {
+                for (window_id, window_weak) in windows.borrow().iter() {
+                    if window_weak.upgrade().map_or(false, |window| window.take_pending_redraw()) {
+                        if let Err(insert_pos) =
+                            windows_with_pending_redraw_requests.binary_search(window_id)
```

Comment:
I don't think we need to do the binary search here, because we clear the vector before.

Reply:
Nevermind, forget the comment about `BTreeMap`, it doesn't really help much (just for this one vector, not for the other).

Reply:
ah right, of course.